keytree/src/lib.rs

```diff
@@ -174,25 +173,18 @@ mod tests {
         let mut rng = ChaChaRng::from_seed(seed);
         let xprv = Xprv::random(&mut rng);
         let xprv_bytes = xprv.to_bytes();
+        let xprv_hex = hex::encode(&xprv_bytes[..]);
```

Comment:
My personal preference would be to not create extra variable for something mundane as convert-to-hex, and inline it inside `assert_eq!`, wdyt?

```rust
assert_eq!(
    hex::encode(&xprv.to_bytes()[..]),
    "4a53c3fbbc59970ee5f85af813875dffc13a904a2e53ae7e65fa0dea6e62c9019f07e7be5551387a98ba977c732d080dcb0f29a048e3656912c6533e32ee7aed"
);
```

Reply:
Oh sure, I just missed this one

Reply:
or factor out a `fn to_hex` converter inside `mod test` so it looks like this:

```
assert_eq!(
    to_hex(xprv.to_bytes()),
    "4a53c3fbbc59970ee5f85af813875dffc13a904a2e53ae7e65fa0dea6e62c9019f07e7be5551387a98ba977c732d080dcb0f29a048e3656912c6533e32ee7aed"
);
```

Reply:
Oh! I like this very much.